File: Server/Data/WoahMe.Data.Models/Place.cs
namespace WoahMe.Data.Models
{
    public class Place
    {
        public int Id { get; set; }

        public string ImageSource { get; set; }
        
        public ImageOrientation ImageOrientation { get; set; }

        public string Title { get; set; }

        public string Description { get; set; }

        public int LocationId { get; set; }

        public virtual Location Location { get; set; }

        public string CreatorId { get; set; }

        public virtual User Creator { get; set; }
    }
}

File: Client/WoahMe/app/src/main/java/woahme/teamwork/com/woahme/Storage/Models/Place.java
package woahme.teamwork.com.woahme.Storage.Models;

import com.orm.SugarRecord;

public class Place extends SugarRecord {
    private String title;
    private String imageSource;
    private String imageOrientation;

    public Place() {
    }

    public Place(String title, String imageSource, String imageOrientation) {
        this.title = title;
        this.imageSource = imageSource;
        this.imageOrientation = imageOrientation;
    }
}